WordPress/Classes/Utility/Logging/CustomLogFormatter.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import CocoaLumberjack
+
+class CustomLogFormatter: NSObject, DDLogFormatter {
+
+    let logTimeStampFormatter: DateFormatter = {
+        let formatter           = DateFormatter()
+        formatter.locale        = Locale(identifier: "en_US_POSIX")
+        formatter.dateFormat    = "yyyy-MM-dd HH:mm:ss:SSS"
+        formatter.timeZone      = TimeZone.current
+        return formatter
+    }()
+
+    func format(message logMessage: DDLogMessage) -> String? {
+        return ("\(logTimeStampFormatter.string(from: logMessage.timestamp)) \(logMessage.message)")
```

Comment:
The nested function call in this string is a bit of a mouthful. Any interest in breaking it into a few lines to end up with something like `"\(time) \(message)"` ?